e2etest/pkg/mac/mac.go

```diff
@@ -105,3 +117,69 @@ func UpdateNodeCache(nodes []corev1.Node, exec executor.Executor) error {
 
 	return nil
 }
+
+// AddrResolutionReq does an ARP/NS request for the given IP.
+func AddrResolutionReq(ip string, exc executor.Executor) error {
+	var cmd string
+	var args []string
+
+	netIP := net.ParseIP(ip)
+	if netIP == nil {
+		return fmt.Errorf("failed to convert %s to net.IP", ip)
+	}
+
+	iface, err := ifaceForIPNetwork(netIP, exc)
+	if err != nil {
+		return err
+	}
+
+	if netIP.To4() != nil {
+		path, err := exec.LookPath("arping")
```

Comment:
Why is this needed? If arping is in PATH, you should be able to execute it I think

Reply:
Isn't needed was a leftover, removed